/src/debversion/debian/patches/drop-sqlx b/src/debversion/debian/patches/drop-sqlx
deleted file mode 100644
index 6aa375172..000000000
--- a/src/debversion/debian/patches/drop-sqlx
+++ /dev/null
@@ -1,22 +0,0 @@
-Index: debversion/Cargo.toml
-===================================================================
---- debversion.orig/Cargo.toml
-+++ debversion/Cargo.toml
-@@ -36,12 +36,6 @@ optional = true
- version = "1"
- optional = true
-
--[dependencies.sqlx]
--version = "0.7"
--features = ["postgres"]
--optional = true
--default-features = false
--
- [dev-dependencies.pyo3]
- version = ">=0.17"
- features = ["auto-initialize"]
-@@ -55,4 +49,3 @@ default-features = false
- default = []
- python-debian = ["dep:pyo3"]
- serde = ["dep:serde"]
--sqlx = ["sqlx/postgres"]
